src/tests/java/picard/illumina/CheckIlluminaDirectoryTest.java

```diff
@@ -51,28 +51,30 @@ private void setUp() throws Exception {
         illuminaDir = IOUtil.createTempDir("ift_test", "IlluminaDir");
 
         interopDir = new File(illuminaDir, "InterOp");
-        if (!interopDir.mkdir()) {
+        if (!interopDir.exists() && !interopDir.mkdir()) {
```

Comment:
Because it is parallelizing on the data provider you have multiple instances of the same method potentially running at the same time. The temp dir is set up before the method not before each invocation of the method with different data providers.

 Perhaps we should turn of data provider parallel execution until we know are tests are constructed in a manner that makes them more easily parallelizable?